### ISSUE TITLE
Unable to Sync

### ISSUE THREAD

**Issue by jarmstrongc4** [DWilliames/google-sheets-sync-figma#31]
While I can fetch data from the [Google Sheet](url removed)
), I'm unable to synch it to Figma. The plugin enters an infinite processing state.
![image](https://user-images.githubusercontent.com/80972323/111806175-54dad980-8897-11eb-9417-2901f5bb4b8e.png)
![image](https://user-images.githubusercontent.com/80972323/111806210-602e0500-8897-11eb-9cb0-106dd514ac3c.png)
![image](https://user-images.githubusercontent.com/80972323/111806241-68864000-8897-11eb-9b11-8b7e8abcd91d.png)

**Comment by prakashkarkee** [comment]
I wish i could help you but it so weird it took long time to processing. If you are okay to share your figma file then i can dig inside what is causing the real issue.  Looking at screenshot seems to be complicated, couldn't find any clue.

**Comment by jarmstrongc4** [comment]
This might be an issue of missing fonts, though it's odd that I had replaced indicated missing fonts and I am able to edit the text in the file. But I'm seeing some strange there and gone action with the A? icon in the upper right of the file indicating missing fonts (but, when I click on it, nothing happens). Even after seemingly addressing this issue, the problem remains. What's very strange is that the plugin works fine in other files, just not this one for some reason.

**Comment by jarmstrongc4** [comment]
I've shared a Figma file where I'm unable to get the plugin to work @PrakTech

**Comment by prakashkarkee** [comment]
Talking about missing fonts, the owner might **locally install** font or may not share **font style**.
According to [documentation](https://help.figma.com/hc/en-us/articles/360039956994-Manage-Missing-Fonts)
> If a collaborator has used a font that's not accessible to you, you will receive a missing font alert.

I have got your google sheet file link only. Let me share my figma file,how it look like,  below is the link.
figma link : https://www.figma.com/file/ZFTQCNPAwfqUjszGBusZ13/Untitled?node-id=25%3A0 
sheet link : https://docs.google.com/spreadsheets/d/1Y303TfENv612wWNsua1YSpzBgV5C5_0z1PT_zVfiaOo/edit?usp=sharing

**Comment by jarmstrongc4** [comment]
I sent an invite to our Figma file by email on Friday.

**Comment by PrakTech** [comment]
If you have sent me invitation then I don't have figma account with
this email address. I use ***@***.*** for figma account.

**Comment by prakashkarkee** [comment]
I use cooldudelove4u@gmail.com for figma account, you might have sent invitation to other email address which don't have account in figma.

**Comment by jarmstrongc4** [comment]
Thanks, I've sent you an invite at that address.

**Comment by prakashkarkee** [comment]
Yeah i get it but i am not able to run the plugin google sync sheet because it is in **can view** mode. Can you change that mode into **can edit** in share icon on top right corner. With your permission, i will work on duplicate of that figma file so that original file get unaffected by my modification.

**Comment by jarmstrongc4** [comment]
Sorry, done

**Comment by jarmstrongc4** [comment]
Direct link to a relevant frame https://www.figma.com/file/FcSuuoBZ6jiu7juNRUYKTL/Rebank-iOS-Wireframes?node-id=0%3A1925 (if helpful)

**Comment by prakashkarkee** [comment]
It took all day long to checkout. Organize the layer then it won't take long time to fetch.
I created page called **cool biometric test** in the document's figma file , below is link.  Its work.
https://www.figma.com/file/oeiuQnvtpTAV6zP7NqQndI/Rebank-iOS-Wireframes-Copy?node-id=307%3A146257

**Additional info** : 
You can check out if this helpful https://www.figma.com/file/ZFTQCNPAwfqUjszGBusZ13/Untitled?node-id=100%3A207 
Check **Completed sample**, **try yourself 1** page. Here I try to illustrate how **Google Sheets Sync** unlocks a whole new possibility of automating your designs  reducing manual duplicating by hand and populating our data/layer/component/variant .

**Comment by jarmstrongc4** [comment]
Thanks for your help! I really appreciate it and will take a closer look at your documentation.

**Comment by prakashkarkee** [comment]
I am glad to help you.
I would like to thank @DWilliames  who is the sole developer and owner of the **Google sheets sync** .In addition, he has really nice documentation about the plugin.
Enjoy the plugin and support it.
